### DISCUSSION TITLE
How to upgrade to version 1.25.5 of Fortinet FortiGate Firewall Logs in Elastic Integration

### DISCUSSION THREAD

**Discussion by ginobyte** [Security-Onion-Solutions/securityonion#13538] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

16

### Storage for /

100Gb

### Storage for /nsm

500Gb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

We have Fortigate up to 7.2 version, the changelog of Fortinet FortiGate Firewall Logs integration describe to update to support this version

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/12833#discussioncomment-10502294


> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

> https://github.com/Security-Onion-Solutions/securityonion/discussions/1720

> Try running soup 'Fortinet FortiGate Firewall Logs' integration should be at version 1.25.4 on SO 2.4.100

**Reply by ginobyte** [reply]
Ok thanks, now I've tha same problem with PAN (PaloAltoNetwork) integration, with the current version on SO 2.4.100 it's at 3.26.2 but in elastic repository it's at 4.0.3. After week-end reboot the SO the PANW dashboard looks empty and there are pipeline error in document event "syslog failed to process field "message": parsing error at position 35: unexpected EOF, Provided Grok expressions do not match field value: [<14>Sep 17 10:07:50"
![Uploading 2024-09-17 12_19_39-Window.jpg…]()

**Reply by ginobyte** [reply]
thanks in advance for your help

**Reply by reyesj2** [reply]
Looks like you may have tried adding a screenshot, but I am unable to see it (just links back to this discussion).

Can you share of screenshot of your elastic fleet dashboard?

**Comment by ginobyte** [comment]
![2024-09-17 12_19_39-Window](https://github.com/user-attachments/assets/a0a9f424-e2ea-423c-84bd-f1cbb979bac2)

**Reply by ginobyte** [reply]
![2024-09-18 09_15_29-Discover_ PAN-OS Flows  Logs PANW  - Elastic](https://github.com/user-attachments/assets/31700747-770d-4649-89bd-69db3df55379)
